src/main/java/uk/gov/companieshouse/api/accounts/model/rest/directorsreport/Director.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.api.accounts.model.rest.directorsreport;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.LocalDate;
+import uk.gov.companieshouse.api.accounts.model.rest.RestObject;
+
+public class Director extends RestObject {
```

Comment:
We'll need a `@JsonInclude(JsonInclude.Include.NON_NULL)` annotating this class. This prevents serialization of nulls.